app/graphql/resolvers/freak_create_resolver.rb

```diff
@@ -6,12 +6,15 @@ class FreakCreateResolver < Resolvers::Base
     argument :last_name, String, required: true
     argument :description, String, required: true
     argument :email, String, required: true
+    argument :norm_id, Integer, required: true
```

Comment:
For norm_id i think we should use `GraphQL::Types::ID` instead of `Integer`.
We should apply this rule everywhere we have an id.